_layouts/default.html

```diff
@@ -26,5 +27,6 @@
 <script>prettyPrint();</script>
 {% endif %}
 <script>JasmineDocs.init();</script>
+<script>anchors.add('.article h1:not(.no_toc),.article h2:not(.no_toc),.article h3:not(.no_toc),.article h4:not(.no_toc)');</script>
```

Comment:
I would like to see this move into `JasmineDocs.init()`, unless there is a good reason to have it separate.

Reply:
Makes sense! Moved into init script.